### DISCUSSION TITLE
r/2024-rv-travel-camping-and-vanlife

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#93] [Announcements]
# r/2024-rv-travel-camping-and-vanlife

12个月近20次旅行，足迹达4省18市，总行程2万余公里

https://blog.ops-coffee.cn/r/2024-rv-travel-camping-and-vanlife

**Comment by thend03** [comment]
介个就是生活，赞美生活的气息

**Comment by tomlinux** [comment]
老哥，有没有运维工作推荐一个

**Comment by yjsdaniel** [comment]
短视频，头条之类的推荐算法这一年劣化的很厉害，充斥着低水平内容和AIGC

**Comment by xyzbz** [comment]
真不错哈，看了一年了、

**Comment by yexia553** [comment]
不知道现在开始新的工作了吗？

**Comment by SeanKChan** [comment]
同洛阳老乡，13年毕业后北漂，混迹大厂近7年，目前在郑州某银行科技条线工作，工作之余，特别喜欢看老哥的游记，充满了人世间的阳光和对美好生活的向往

**Reply by ops-coffee** [reply]
非常棒！
